src/walletSdk/Horizon/Transaction/TransactionBuilder.ts

```diff
@@ -90,6 +90,35 @@ export class TransactionBuilder extends CommonTransactionBuilder<TransactionBuil
     return this;
   }
 
+  pathPay(
+    destinationAddress: string,
+    sendAsset: StellarAssetId,
+    destAsset: StellarAssetId,
+    amount: string,
+    destMin?: string,
```

Comment:
imo `destMin` can be a bit confusing when first being introduced to path payments. So leaving it optional and using the smallest value possible as default if not given

Reply:
Can you add doc describing what destMin is? You can copy it from [our documentation ](https://developers.stellar.org/docs/encyclopedia/path-payments)